creator/settings/production.py

```diff
@@ -295,15 +295,14 @@ def traces_sampler(sampling_context):
         },
     },
     "loggers": {
-        "TaskLogger": {},
+        "TaskLogger": {"handlers": ["task"], "level": "INFO"},
         "graphql.execution.utils": {
             "handlers": ["command"],
             "level": "CRITICAL",
         },
         "creator": {"handlers": ["task"]},
         "creator.management": {"handlers": ["command"], "level": "INFO"},
         "rq.worker": {"handlers": ["rq_console"]},
-        "creator.decorators": {"handlers": ["task"]},
```

Comment:
Why remove this? Don't you still want some of those log statements in the task decorator app code to show up in cloudwatch?